### PR TITLE
Add sticky positioning demo

### PR DESCRIPTION
**Pull request by jessmitch42**
Daily Prebuilt demo that shows how the CSS `position:sticky;` property can be used to keep the call on screen even when the user scrolls down and the call would otherwise scroll off screen. 

Written in HTML/JS/CSS. Intentionally simple (didn't spend a ton of time on the CSS) because I don't want the actual focus (two lines of CSS!) to get lost in a sea of code.

As an addition to make it a little more interesting, the video call also shrinks on scroll so it takes up less space if the user is reading the content below the call.

### CODE REVIEW

File: samples/daily-prebuilt/sticky-positioning/index.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    <script src="index.js"></script>
+  </head>
+  <body>
+    <main>
+      <h1>Daily Prebuilt - Sticky positioning example</h1>
+
+      <h2>Enter a Daily room URL to join a Daily call</h2>
+      <form id="joinForm">
+        <label for="url">Room URL</label>
+        <input id="url" type="text" />
```
Comment (lazeratops) [inline samples/daily-prebuilt/sticky-positioning/index.html:18]: I know we don't really want to do any validation here for simplicity, but could we just mark this field as required and make the type "url" to at least have the default checks that the URL is filled in and is at least a URL?

**Review by lazeratops — COMMENTED** [review]
Just _one_ suggestion, and otherwise this lgtm!

File: samples/daily-prebuilt/sticky-positioning/index.html (hunk 1 above)
Comment (lazeratops) [reply samples/daily-prebuilt/sticky-positioning/index.html:18]: (Actually I cannot remember if I also mentioned this in the outline and you had a reason for not including that kind of validation here... sorry if I did and just forgot!)

Comment (jessmitch42) [reply samples/daily-prebuilt/sticky-positioning/index.html:18]: No reason other than to keep this supppper basic, but I can definitely update the input with those changes!

Comment (lazeratops) [reply samples/daily-prebuilt/sticky-positioning/index.html:18]: Let's go for it! I think a couple of extra HTML attributes won't complicate it much and they'll provide significant benefit

Comment (jessmitch42) [reply samples/daily-prebuilt/sticky-positioning/index.html]: Updated 🚀

**Comment by jessmitch42** [comment]
hey @lazeratops little reminder for a final pr review :)

**Review by lazeratops — APPROVED** [review]
LGTM!

**Pull request merged**